File: iniciante/python/1020-idade-em-dias.py
def idade_em_dias(d):
    dias = {'ano(s)': 365, 'mes(es)': 30, 'dia(s)': 1}
    dias_restantes = d
    for ano_mes_dia, n_dias in dias.items():
        dia = dias_restantes // n_dias
        dias_restantes -= dia * n_dias
        print(f'{dia} {ano_mes_dia}')


total_dias = int(input())
idade_em_dias(total_dias)

File: iniciante/c/1020-idade-em-dias.c
#include <stdio.h>

int main(){
    int anos, meses, dias, n;
    
    scanf("%d", &n);
    
    anos = n / 365;
    meses = (n - anos * 365) / 30;
    dias = n - anos * 365 - meses * 30;
    
    printf("%d ano(s)\n", anos);
    printf("%d mes(es)\n", meses);
    printf("%d dia(s)\n", dias);

    return 0;
}
